modules/lofreq/1.0/lofreq.smk

```diff
@@ -54,51 +54,89 @@ rule _lofreq_run:
         fasta = reference_files("genomes/{genome_build}/genome_fasta/genome.fa"),
         dbsnp = reference_files("genomes/{genome_build}/variation/dbsnp.common_all-151.vcf.gz")
     output:
-        vcf_snvs = CFG["dirs"]["lofreq"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/somatic_final_minus-dbsnp.snvs.vcf.gz",
-        vcf_indels = CFG["dirs"]["lofreq"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/somatic_final_minus-dbsnp.indels.vcf.gz"
+        vcf_snvs_filtered = CFG["dirs"]["lofreq"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/somatic_final_minus-dbsnp.snvs.vcf.gz",
+        vcf_indels_filtered = CFG["dirs"]["lofreq"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/somatic_final_minus-dbsnp.indels.vcf.gz",
+        vcf_snvs_all = CFG["dirs"]["lofreq"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/somatic_final.snvs.vcf.gz",
+        vcf_indels_all = CFG["dirs"]["lofreq"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/somatic_final.indels.vcf.gz"
     log:
         stdout = CFG["logs"]["lofreq"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/lofreq.stdout.log",
         stderr = CFG["logs"]["lofreq"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/lofreq.stderr.log"
     params:
         opts = CFG["options"]["lofreq"],
-        regions = op.switch_on_wildcard("seq_type", CFG["switches"]["regions_bed"])
+        regions = op.switch_on_wildcard("seq_type", CFG["switches"]["regions_bed"]),
+        rm_files = CFG["dirs"]["lofreq"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/normal_relaxed.vcf.gz"
     conda:
         CFG["conda_envs"]["lofreq"]
     threads:
         CFG["threads"]["lofreq"]
     resources:
-        mem_mb = CFG["mem_mb"]["lofreq"],
-        bam =1
+        **CFG["resources"]["lofreq"]
     shell:
         op.as_one_line("""
         lofreq somatic {params.opts} --threads {threads} -t {input.tumour_bam} -n {input.normal_bam}
-        -f {input.fasta} -o $(dirname {output.vcf_snvs})/ -d {input.dbsnp} {params.regions} 
+        -f {input.fasta} -o $(dirname {output.vcf_snvs_filtered})/ -d {input.dbsnp} {params.regions} 
         > {log.stdout} 2> {log.stderr}
+            &&
+        rm {params.rm_files}
```

Comment:
Kostia, what's the rationale for an explicit deletion command, rather than marking the file as a temp output file?

Reply:
I did in that way, because I thought it will make easier to add here other files, too, if we will decide to clear other output files later. Otherwise, I was thinking I will over complicate output of the rule.